translations/messages.nl.yml

```diff
@@ -1,111 +1,32 @@
-service_provider: Service provider
+# Registration
 
-page.title: GSSP SDK
+registration.sub_title: Registration
+status_registration_initial: Registreer met jou security key.
 
-page.header.stepup-gateway-gssp: Wiki
-page.header.stepup-gssp-bundle: GSSP-bundel
+# Authentication
 
-page.navigation.route.homepage: Home
-page.navigation.route.app_identity_registration: Registratie
-page.navigation.route.app_identity_authentication: Authenticatie
-page.navigation.route.gssp_saml_sso: SSO
-page.navigation.route.gssp_saml_sso_return: SSO return
-page.navigation.route.gssp_saml_metadata: Metadata
-page.navigation.route.sp_demo: Demo SP
+authentication.sub_title: Authenticeren
+status.authentication_initial: Authenticeer met jou security key.
 
-page.footer.terms: Algemene voorwaarden
+# General status messages
 
-page.index.title: Welkom bij Generieke SAML Stepup Provider SDK
-page.index.body.intro: |
-    De applicatie is klaar voor gebruik. Dit voorbeeldproject kan gebruikt worden om een eigen implementatie van een GSSP verificatiemiddel te ontwikkelen.
-    Het demonstreert registratie en authenticatie van second factor tokens geïnitieerd door een dummy service provider.
+status.general_error: Oeps! Er is iets fout gegaan, probeer het opnieuw.
+status.webauthn_not_supported: Webauthn wordt niet ondersteund door uw browser.
```

Comment:
Previously you used je/jij/jouw, and now the more formal U. Ik think we always use the less formal options.